app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationScreen.kt

```diff
@@ -210,8 +212,9 @@ private fun CompleteRegistrationContent(
         )
         Spacer(modifier = Modifier.height(8.dp))
         PasswordStrengthIndicator(
-            modifier = Modifier.padding(horizontal = 16.dp),
```

Comment:
I assume this was just a small design change?

Reply:
yeah extra padding that I missed in the other PR